tools/redis-api.hbs

```diff
@@ -37,13 +37,25 @@ public interface RedisAPI {
   }
 
 {{#each .}}
+  /**
+   * Redis command <a href="https://redis.io/commands/{{ name }}">{{ camelcase safename }}</a>.
+   * @return fluent self
+   */
   @Fluent
   default RedisAPI {{ camelcase safename }}({{{ types }}}{{#if argLen}}, {{/if}}Handler<AsyncResult<@Nullable Response>> handler) {
     send(Command.{{ enum }}{{#if argLen}}, {{{ args }}}{{#if variable}}.toArray(){{/if}}{{/if}}).setHandler(handler);
     return this;
   }
+
+  /**
+   * Redis command <a href="https://redis.io/commands/{{ name }}">{{ camelcase safename }}</a>.
+   * @return Future response.
+   */
+  default Future<@Nullable Response> {{ camelcase safename }}({{{ types }}}) {
+    return send(Command.{{ enum }}{{#if argLen}}, {{{ args }}}{{#if variable}}.toArray(){{/if}}{{/if}});
+  }
 {{/each}}
 
   @GenIgnore
```

Comment:
should't we remove @GenIgnore ?

Reply:
we can't this a `varargs` method which is handy to delegate all synthetic API calls as the compiler enforces the types

Reply:
oh right, we need to support varargs at some point in codegen :-)